test/civ2.test.js

```diff
@@ -0,0 +1,699 @@
+    });
+
+    it('registers a link update command', () => {
+      expect(robot.hear.getCall(14).args[0].toString()).to.equal(/update links (\S*)/.toString());
+    });
+
+    it('registers a instance update command', () => {
+      expect(robot.hear.getCall(15).args[0].toString()).to.equal(
+        /update instance (\S*)( on (\S*) environment)?( to version (\S*))?/.toString()
+      );
+    });
+
+    it('registers a public release listener', function() {
+      expect(robot.hear.getCall(16).args[0].toString()).to.equal('/Publicly release (\\S*)/');
+    });
+  });
+
+  describe('Commands', () => {
+    let civ2mock;
+    let robot;
+
+    beforeEach(() => {
+      hubotMock = new HubotMock();
+      civ2mock = new CIV2Mock();
+      robot = new Robot(hubotMock);
+    });
+
+    afterEach(() => civ2mock.restore());
+
+    describe('Chat', () => {
+      describe('Release stoic', () => {
+        it('should call civ2.release', async () =>
+          expectCiv2CommandCall(
+            'release stoic 123',
+            [{ command: 'release', expectedArgs: ['123', true] }],
+            [{ method: 'reply', args: ['Release in progress.'] }]
+          ));
+
+        it('should notify in an error occurs', async () =>
+          expectCiv2Error('release stoic 123', [{ command: 'release', expectedArgs: ['123', true] }]));
+      });
+
+      describe('Rollback stoic', () => {
+        it('should call civ2.release', async () =>
+          expectCiv2CommandCall(
+            'rollback stoic 123',
+            [{ command: 'release', expectedArgs: ['123', false] }],
+            [{ method: 'reply', args: ['Rollback in progress.'] }]
+          ));
+
+        it('should notify in an error occurs', async () =>
+          expectCiv2Error('rollback stoic 123', [{ command: 'release', expectedArgs: ['123', false] }]));
+      });
+
+      describe('Create feature instance', () => {
+        it('should call civ2.createFeatureCluster', async () =>
+          expectCiv2CommandCall(
+            'create feature instance toto/pipo',
+            [{ command: 'createFeatureCluster', expectedArgs: ['toto/pipo'] }],
+            [{ method: 'reply', args: ['Creation in progress.'] }]
+          ));
+
+        it('should notify in an error occurs', async () =>
+          expectCiv2Error(
+            'create feature instance toto/pipo',
+            [{ command: 'createFeatureCluster', expectedArgs: ['toto/pipo'] }],
+            'send'
+          ));
+      });
+
+      describe('Destroy feature instance', () => {
+        it('should call civ2.destroyFeatureCluster', async () =>
+          expectCiv2CommandCall(
+            'destroy feature instance toto/pipo',
+            [{ command: 'destroyFeatureCluster', expectedArgs: ['toto/pipo'] }],
+            [{ method: 'reply', args: ['Destruction in progress.'] }]
+          ));
+
+        it('should notify in an error occurs', async () =>
```

Comment:
You should add a curly bracket at the start of the arrow. It would bring more consistency to this file, and it's often easier to read mocha tests when they all use blocks, as our eyes are very used to this pattern.